config/redirects.txt

```diff
@@ -116,4 +115,4 @@
 ^\/reference\/OS\/overview\/2.x\/?(\?.+)?(#.+)?$ -> /reference/OS/overview/$1$2
 ^\/reference\/OS\/network\/1.x\/?(\?.+)?(#.+)?$ -> /reference/OS/network/$1$2
 ^\/reference\/OS\/network\/2.x\/?(\?.+)?(#.+)?$ -> /reference/OS/network/$1$2
-^\/learn\/manage\/app-types\/?(\?.+)?(#.+)?$ -> /learn/manage/fleet-types/$1$2
\ No newline at end of file
+^\/learn\/manage\/app-types\/?(\?.+)?(#.+)?$ -> /learn/manage/fleet-types/$1$2
```

Comment:
Not sure what change was made to this line?

Reply:
The change was whitespace added at the end of the line. Fixed in 6d8c525.